lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -190,6 +194,7 @@ if (typeof module !== 'undefined' && module.exports) {
 /**
  * @typedef {{
  *     type: string,
+ *     styleAs: (string|undefined),
```

Comment:
> I didn't see it as table-only we just didn't have a need for text styling yet

only the table renderer looks for it and only table css exists to support it, so I think it makes sense to make it clear it's limited there until someone comes along and adds broader support.